src/scenes/AcceleratorRouter/ParticipantProjects/Reports/ReportView.tsx

```diff
@@ -200,6 +238,7 @@ const ReportView = () => {
 
   return (
     <>
+      {showPublishModal && <PublishModal onClose={() => setShowPublishModal(false)} onSubmit={publishReport} />}
```

Comment:
Is the `open` prop in the `confirm` component essentially the same thing as doing `{showPublishModal && <...`? Looks like we've done it both ways so no need to change, just curious

Reply:
We initially used the "open" prop, but because the state does not clear if you close the modal we moved to the other approach. With the `{showPublishModal && <...` it's a new render every time so if you close the modal and open again, there will be no old data. In this case we don't have fields to fill, so it would be the same

Reply:
Ah that makes sense!